internal/provider/models/sources/test/datadog_test.go

```diff
@@ -20,9 +20,7 @@ func TestDatadogSource(t *testing.T) {
 					resource "mezmo_pipeline" "test_parent" {
 							title = "parent pipeline"
 						}`) + `
-					resource "mezmo_datadog_source" "my_source" {
-						signing_key = "nope"
-					}`,
+					resource "mezmo_datadog_source" "my_source" {}`,
```

Comment:
Should we delete the parameter from `internal/provider/models/sources/datadog.go` too?

Reply:
it's not actually part of that model, looks like just copypasta

Reply:
oh! nice cleanup!